app/views/users/index.html.erb

```diff
@@ -7,16 +6,25 @@
     <th>Email Address</th>
     <th>Phone Number</th>
     <th>Join Date</th>
+    <th>Status</th>
+    <th>Change Status</th>
   </tr>
 
-  <% @users.each do |user| %>                    
+  <% @users.each do |user| %>
     <tr>
       <td><%= user.full_name %></td>
       <td><%= user.email %></td>
       <td><%= user.phone_number %></td>
       <td><%= user.created_at %></td>
-    </tr>
-  <% end %>
 
-  <%= button_to "Export to CSV", "/users.csv", :method => :get%>
-</table>
+        <% if user.deactivated? %>
+        <td> Inactive </td>
+        <td> <%= link_to "Activate User", deactivate_index_path(id: user.id), method: :post, data: { confirm: "Are you sure you want to Activate this user?" } %> </td>
```

Comment:
Maybe `User` word is not necessary here, admin already knows that he is on the user table, maybe just `Activate and Deactivate`